Zs branch 01

ZdravekSprite opened the pull request:
sync with ohjelmointi-20-master

S4ndyk commented:
Hi! It looks like apart from spelling errors you've also deleted some of the old comments in your commits. We welcome spelling corrections but would like if you could leave the comments etc. as they are. If you fix them i will merge this.

ZdravekSprite commented:
I don't know which comments you mean, because I even added some original
Finnish parts because someone who translated them deleted them before, and
there are differences in parts in Finnish because I updated with content
from https://github.com/rage/ohjelmointi -20 which is, at least I assumed,
more current.

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Mon, Jun 15, 2020 at 1:11 PM Santtu Lempinen <notifications@github.com>
wrote:

> Hi! It looks like apart from spelling errors you've also deleted some of
> the old comments in your commits. We welcome spelling corrections but would
> like if you could leave the comments etc. as they are. If you fix them i
> will merge this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rage/java-programming/pull/145#issuecomment-644064164>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMM7KACZJZZ3ZRNIL4TGWTLRWX6VJANCNFSM4M2KTTUA>
> .
>
